uniffi_bindgen/src/backend/config.rs

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use serde::{Deserialize, Serialize};
+
+/// Config value for wrap/unwrap
+///
+/// These define an expression to do the conversion, with `{}` as a placeholder for the value being
+/// wrapped/unwrapped.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+pub struct WrapExpression(String);
+
+impl WrapExpression {
+    pub fn render(&self, var: &str) -> String {
+        self.0.replace("{}", var)
+    }
+}
```

Comment:
Do I understand correctly: this is called `WrapExpression` because it's used for wrapping and unwrapping? How about naming it for what it does (e.g. `TemplateExpr`)?